bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocationProvider.java

```diff
@@ -23,4 +26,10 @@
      */
     PointType getLocation();
 
+    /**
+     * Provides access to the time zone property
+     *
+     * @return the time zone set in the Paper UI and if there isn't one, the default time zone of the underlying system
+     */
+    TimeZone getTimeZone();
```

Comment:
Shouldn't there be a `TimeZoneProvider` interface that is implemented by `I18nProviderImpl` similar to `LocaleProvider` and `LocationProvider`?